init.groovy.d/ldap.groovy

```diff
@@ -22,6 +21,7 @@ Thread.start {
     sleep 10000
 
     if (ldap_server && (ldap_rootDN || ldap_inhibitInferRootDN)) {
```

Comment:
ldap_inhibitInferRootDN would be NULL here which makes this logic useless and would be dangerous.